M2/cmake/FindFFI.cmake

```diff
@@ -1,3 +1,6 @@
+################### This file is made obsolete by the PR 2968 #######################
+# This is public domain code
```

Comment:
Are we sure this is true?  My guess is that it has an Apache license since it's included in the LLVM source (see https://github.com/llvm/llvm-project/blob/main/llvm/cmake/modules/FindFFI.cmake)

Reply:
not %100 sure, no. It's good to dig it up

Reply:
I forgot where I saw this, but CMake FindModules are always under the lightest license (after all, it's barely code ... and who would want to copyright code for _findin_ something open source)